smother/control.py

```diff
@@ -81,6 +86,13 @@ def write(self, file_or_path, append=False, timeout=10):
         multiprocess test environment.
         """
         if isinstance(file_or_path, six.string_types):
+            if self.coverage and self.coverage.config.parallel:
```

Comment:
my first instinct is that `write()` should be "dumb" and treat file_or_path as litearally as possible. Can we move this logic up to a helper function invoked by the caller of `write`? Or do you think it makes more sense here?